src/components/Presentation.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, Fragment } from 'react'
+import './Styles/Presentation.css'
+import  phone_showcase  from '../img/phone showcase.png';
+
+
+const Presentation = () => {
+    const [colorChange, setColorchange] = useState(false);  
+    const changeSectionColor = () =>{
+       if(window.scrollY >= 30){
+         setColorchange(true);
+       }
+       else{
+         setColorchange(false);
+       }
+    }
+     window.addEventListener('scroll', changeSectionColor);
+
+    return (
+      <Fragment>
+        <div className = {colorChange ? 'section colorChange' : 'section'}>
```

Comment:
```suggestion
        <div className = { colorChange ? 'section colorChange': 'section' }>
```
En Javascript por lo general se deja un espacio en blanco dentro de las llaves y en el operador ternario no se deja espacio entre los dos puntos y el segundo operando